cgroup/cgroup_parser_darwin.go

```diff
@@ -5,7 +5,11 @@
 
 package cgroup
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups"
```

Comment:
Needed for the L22 return type, though admittedly I dont know why this wasn't erroring for us locally